### PR TITLE
use kwargs as sugested in issue #47

### PR DESCRIPTION
**Pull request by gawel**
Make it much more easier to maintain. Any objection ?

### CODE REVIEW

File: django_webtest/__init__.py [hunk 1]
```diff
@@ -129,77 +133,41 @@ def is_redirect(r):
 
         return response
 
-    def post(self, url, params='', headers=None, extra_environ=None,
```
Comment (kharandziuk) [inline django_webtest/__init__.py]: Hi @gawel there was a discussion about kwargs https://github.com/django-webtest/django-webtest/pull/23#discussion_r8992561
I guess @kmike should merge this commit

**Comment by kmike** [comment]
I wrote explicit arguments in first place to enable autocompletion in IDEs.

**Comment by gawel** [comment]
Hm ok... I don't care about IDEs.

Another option is to introspect webtest's method and generate new methods on the fly with correct arguments. but this mean a lot of «magic»...

**Comment by kharandziuk** [comment]
@django-webtest/owners Hey, if nobody can propose a better solution I will merge this PR

**Comment by inglesp** [comment]
I've (independently) made the same change in #62.  See 518b76e0fdde8e7b2c65f29dfedb1d531e33e568 and 9e316cd333ffdf1ff968e8103619bd4c6d005355 for my justification.

I can see the argument that using kwargs makes things slightly harder for IDE users, but I think this is more than offset by (a) making it easy to support new options in WebTest, and (b) the difficulty of keeping large numbers of kwargs in sync.

**Pull request closed without merging**